<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>System Utility Dashboard</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <widget class="QTabWidget" name="tabs">

      <widget class="QWidget" name="tabSystem">
       <attribute name="title">
        <string>System Info</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_1">
        <item>
         <widget class="QPushButton" name="btnRefreshSystem">
          <property name="text">
           <string>Refresh System Info</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="txtSystemInfo">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

      <widget class="QWidget" name="tabProcesses">
       <attribute name="title">
        <string>Processes</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btnListProcesses">
          <property name="text">
           <string>List Processes</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="lstProcesses"/>
        </item>
       </layout>
      </widget>

      <widget class="QWidget" name="tabBackup">
       <attribute name="title">
        <string>Backup Tool</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLineEdit" name="inputBackupFolder">
          <property name="placeholderText">
           <string>Folder to backup...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnBackup">
          <property name="text">
           <string>Create Backup</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="txtBackupResult">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

      <widget class="QWidget" name="tabNetwork">
       <attribute name="title">
        <string>Network Tools</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="inputPing">
          <property name="placeholderText">
           <string>Enter host to ping...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPing">
          <property name="text">
           <string>Ping Host</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="txtPingResult">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

      <widget class="QWidget" name="tabLogs">
       <attribute name="title">
        <string>Log Viewer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLineEdit" name="inputLogFile">
          <property name="placeholderText">
           <string>Path to log file...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLoadLog">
          <property name="text">
           <string>Load Log File</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="txtLogContent">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>

     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>